tests/unit_and_functional/conftest.py

```diff
@@ -75,3 +77,22 @@ def k8s_env_vars() -> Iterable[bool]:
     finally:
         yield True
     del os.environ['KUBERNETES_SERVICE_HOST']
+
+def onerror(func, path, exc_info):
```

Comment:
`on_error` would be better, despite the naming convention used in the Python standard library (which is not always PEP8, due to it being written before PEP8...).

Reply:
👍